config_defaults_inc.php

```diff
@@ -1955,6 +1955,28 @@
  */
 $g_file_upload_method = DATABASE;
 
+/**
+ * Use Zip_attachments: zip attachments in order to optimize  
+ * network performance and save disk/db space.
+ * @global integer $g_zip_attachments
+ */
+$g_zip_attachments		=	OFF;
+
+/**
+ * Use Zip_excludes: define the file types to omit from the zipping process 
+ * because the files are already compressed and/or one does not want to
+ * compress a specific file type.
+ * Values should be provided in lowercase.
+ * @global integer $g_zip_excludes
+ */
+$g_zip_excludes			= array('pdf','zip','docx','xlsx','pptx');
```

Comment:
Rename to $g_zip_attachments_excluded_extensions

Reply:
Adjusted accordingly